languages/csharp-dotnet-2.2/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/opt/algorithm/bin/Release/netcoreapp2.2/__ALGO__.dll", "destination":"/opt/algorithm/algo.dll"},
```

Comment:
Question whether this has to be absolute path, or can we remove `/opt/algorithm/` and use relative paths?

Reply:
@pmcq would love your feedback here, I just copied from the python languages config.json here.

Reply:
i much prefer absolute path but the destination can be anywhere.  the important thing is that there needs to be an entrypoint in `/usr/local/bin/algorithmia-pipe`.  also the `__ALGO__` here isn't going to be templated since this isn't part of the template for an algorithm

Reply:
Oh that's a big deal then. Dotnet creates a dll file with the same name as the incoming project. Would a `/*.dll` work here? Or would that wildcard fail?

Reply:
yeah you should be able to use a wildcard like that  - it just gets translated to a docker copy command here https://github.com/algorithmiaio/langpacks/blob/master/languages/Dockerfile.compile.j2#L10 so anything supported by docker would work